server/control/src/main/java/org/kaaproject/kaa/server/control/service/log/RecordLibraryGenerator.java

```diff
@@ -53,20 +59,24 @@
      * Generate record library.
      *
      * @param logSchemaVersion the log schema version
-     * @param logSchema the log schema
+     * @param logSchema        the log schema
      * @return the record structure library
      * @throws Exception the exception
      */
     public static FileData generateRecordLibrary(int logSchemaVersion, String logSchema) throws Exception {
-        Schema recordWrapperSchema = RecordWrapperSchemaGenerator.generateRecordWrapperSchema(logSchema);
-        List<JavaDynamicBean> javaSources = JavaSdkGenerator.generateSchemaSources(recordWrapperSchema);
+        final Schema recordWrapperSchema = RecordWrapperSchemaGenerator.generateRecordWrapperSchema(logSchema);
+
+        Map<String, Schema> uniqueSchemas = SchemaUtil.getUniqueSchemasMap(new LinkedList<Schema>() {{
```

Comment:
remove this syntax please. Also any modifications to this class are meaningless due to processing of single schema (log) here